frontend/src/lib/DefaultStreamlitEndpoints.test.ts

```diff
@@ -129,6 +137,63 @@ describe("DefaultStreamlitEndpoints", () => {
         )
       )
     })
+
+    it("errors on bad status", async () => {
+      axiosMock
+        .onPost("http://streamlit.mock:80/_stcore/upload_file")
+        .reply(() => [400])
+
+      await expect(
+        endpoints.uploadFileUploaderFile(
+          MOCK_FILE,
+          "mockWidgetId",
+          "mockSessionId"
+        )
+      ).rejects.toEqual(new Error("Request failed with status code 400"))
+    })
```

Comment:
This is a test for the `uploadFileUploaderFile` implementation that I realized was missing from my previous PR.